ISISScientificComputing/autoreduce
# Django orm queue processors

### Summary of work
* Removing old ORM aspects from QueueProcessor

### How to test your work
* Ensure this code is running on develop queue processor machine
* Manually submit runs (feel free to try via the webapp, via manual submission script, or run_detection)
* Ensure that runs are still successfully added to the database

* Code review
* Ensure unit tests pass

### Additional comments
* Note: The test coverage for the queueproc_utils and queue_processor will be handled in a separate PR. This PR is already large and there were not tests for these before so I don't intend to further inflate the PR.
